Sources/Networking/Operations/Handling/CustomerInfoResponseHandler.swift

```diff
@@ -15,15 +15,10 @@ import Foundation
 
 class CustomerInfoResponseHandler {
 
-    let userInfoAttributeParser: UserInfoAttributeParser
-
-    init(userInfoAttributeParser: UserInfoAttributeParser = UserInfoAttributeParser()) {
-        self.userInfoAttributeParser = userInfoAttributeParser
-    }
+    init() { }
```

Comment:
are we not injecting this for tests? how are we testing this class?

Reply:
It wasn't being mocked, so instead of injecting it this PR turned it into an `enum`. But `UserInfoAttributeParser` actually goes away in #1431.